examples/hello/package.json

```diff
@@ -19,7 +18,6 @@
     "typescript": "^5.2.2"
   },
   "dependencies": {
-    "@dbos-inc/dbos-sdk": "../..",
-    "ts-node": "^10.9.1"
```

Comment:
This pulls in the entire Typescript compiler, so it has to go

Reply:
Not clear why this couldn't be a dev dependency.